pandas/core/indexing.py

```diff
@@ -2044,7 +2044,6 @@ def _setitem_single_block(self, indexer, value, name: str) -> None:
                     # e.g. test_loc_setitem_empty_append_expands_rows
                     loc = item_labels.get_loc(col)
                     # Go through _setitem_single_column to get
-                    #  FutureWarning if relevant.
```

Comment:
perhaps the `# Go through _setitem_single_column to get` sentence can be removed too?

Reply:
Good catch. Removed